src/directLine.ts

```diff
@@ -252,6 +261,7 @@ export interface IBotConnection {
     connectionStatus$: BehaviorSubject<ConnectionStatus>,
     activity$: Observable<Activity>,
     end(): void,
+    getConversation(): Conversation,
```

Comment:
`getConversation` shouldn't be part of `IBotConnection`, which is a protocol-agnostic interface and doesn't know what a `Conversation` is. Just expose  `getConversation ` as a public method on DirectLine (which, in addition to implementing `IBotConnection` is free to have other methods and fields),  although I might just suggest skipping that and just exposing the required fields as `public`.